### ISSUE TITLE
Can't disconnect gracefully

### ISSUE THREAD

**Issue by simonwjackson** [f-hj/slsk-client#35]
```js
const slsk = require('slsk-client')

slsk.connect({
  user: 'user',
  pass: 'pass'
}, (err, client) => {
  client.search({
    req: 'random',
    timeout: 2000
  }, (err, res) => {
    if (err) return console.log(err)
    console.log(res)

    /* *** Trying to exit *** */
    client.destroy()
    slsk.disconnect()

  })
})
```

I cant exit the script without <kbd>Ctrl-C</kbd>. Am I doing something wrong here?

**Comment by riteable** [comment]
Is `process.exit()` not an option?

**Comment by simonwjackson** [comment]
I guess so, but It feels a bit hacky to rely on `process.exit()`.

**Comment by f-hj** [comment]
Should be fixed in 1.1.0
